sqlalchemy_continuum/dialects/postgresql.py

```diff
@@ -508,42 +522,54 @@ def sync_trigger(conn,
         set(c.name for c in parent_table.c) -
         set(c.name for c in version_table.c if not c.name.endswith('_mod'))
     )
-    drop_trigger(conn, parent_table.name)
+    drop_trigger(conn, parent_table.name, parent_table.schema)
     create_trigger(conn,
                    table=parent_table,
                    excluded_columns=excluded_columns,
-                   version_table_name_format=version_table_name_format)
+                   versioning_manager=versioning_manager)
 
 
 def create_trigger(
     conn,
     table,
     transaction_column_name='transaction_id',
     operation_type_column_name='operation_type',
-    version_table_name_format=DEFAULT_VERSION_TABLE_NAME_FORMAT,
+    versioning_manager=None,
```

Comment:
Should we require the `versioning_manager` to be passed in?

Reply:
I guess it would be good to, yes - since there are a lot of guards against its existence in this function.  I'll look into that

Reply:
done